disnake/ext/commands/params.py

```diff
@@ -916,7 +986,12 @@ def expand_params(command: AnySlashCommand) -> List[Option]:
         raise TypeError(f"Couldn't find an interaction parameter in {command.callback}")
 
     for injection in injections.values():
-        params += collect_nested_params(injection.function)
+        collected = collect_nested_params(injection.function)
+        if injection.autocompleters is not Ellipsis:
+            for p in collected:
+                if f := injection.autocompleters.get(p.name):
```

Comment:
This should raise a `ValueError` if an autocomplete callback is set for a nonexistent parameter, similar to `InvokableSlashCommand.autocomplete`.

Reply:
Done in [c09073b8ffe6cb971385f63b2cb0602b6ef228e4](https://github.com/DisnakeDev/disnake/pull/675/commits/c09073b8ffe6cb971385f63b2cb0602b6ef228e4)